KingfisherWebP/Classes/CGImage+WebP.m

```diff
@@ -135,26 +127,18 @@ Call WebPDecode() on a multi-frame webp data will get an error (VP8_STATUS_UNSUP
     VP8StatusCode result = WebPDecode(payload, payloadSize, &config);
     if ((result != VP8_STATUS_OK) && (result != VP8_STATUS_NOT_ENOUGH_DATA)) FAIL_CGImageCreateWithWebPData;
 
-    if (iter.x_offset != 0 || iter.y_offset != 0) {
-        void *tmp = calloc(1, destLength);
-        if (tmp) {
-            vImage_Buffer src = {destBytes, canvasHeight, canvasWidth, bytesPerRow};
-            vImage_Buffer dest = {tmp, canvasHeight, canvasWidth, bytesPerRow};
-            vImage_CGAffineTransform transform = {1, 0, 0, 1, iter.x_offset, -iter.y_offset};
-            uint8_t backColor[4] = {0};
-            vImageAffineWarpCG_ARGB8888(&src, &dest, NULL, &transform, backColor, kvImageBackgroundColorFill);
-            memcpy(destBytes, tmp, destLength);
-            free(tmp);
-        }
-    }
+    CGDataProviderRef provider = CGDataProviderCreateWithData(destBytes, config.output.u.RGBA.rgba, config.output.u.RGBA.size, ReleaseDataCallback);
+    destBytes = NULL;
 
-    provider = CGDataProviderCreateWithData(destBytes, destBytes, destLength, ReleaseDataCallback);
-    if (!provider) FAIL_CGImageCreateWithWebPData;
-    destBytes = NULL; // hold by provider
+    CGColorSpaceRef colorSpace = CGColorSpaceCreateDeviceRGB();
```

Comment:
It seems the `colorSpace` created here is no longer needed and should be removed.